dataactcore/scripts/load_historical_certified_dabs.py

```diff
@@ -141,6 +141,14 @@ def insert_file(filename, submission_id, file_type_id, csv_schema, long_to_short
     # Only use the columns needed for the DB table
     data = data.rename(columns=lambda x: x.lower().strip())
     data = data.rename(index=str, columns=long_to_short_dict)
+    # The following columns were added later and need to be accounted for in old data, where they would be blank
+    # If DEFC doesn't exist for files B or C, add it and make them all blank
+    if 'disaster_emergency_fund_code' not in data and file_type_id in (FILE_TYPE_DICT['program_activity'],
+                                                                       FILE_TYPE_DICT['award_financial']):
+        data['disaster_emergency_fund_code'] = ''
```

Comment:
Do we set these to `NULL` later in the code? I'm noticing that we keep them as `null` in the database if not provided.